blog/test-toolkit/src/main/kotlin/com/excref/kotblog/blog/service/test/helper/ServiceImplTestHelper.kt

```diff
@@ -12,5 +13,7 @@ class ServiceImplTestHelper {
     fun buildTag(): Tag = buildTag(UUID.randomUUID().toString())
 
     fun buildTag(name: String): Tag = Tag(name)
+    fun  buildCategory(): Category = buildCategory(UUID.randomUUID().toString())
```

Comment:
I jus found better solution, you can use `default` values for the arguments and avoid second method creation :)
Please refactor tag as well if you can find time for it